### DISCUSSION TITLE
启动成功 登录问题

### DISCUSSION THREAD

**Discussion by JacianZhu** [simple-robot/Discussions#151] [Q&A]
### 涉及的编程语言

Java

### 涉及的组件库

mirai (QQ)

### 疑问描述
![image](https://github.com/simple-robot/Discussions/assets/51085457/1c26ac7a-f660-4daa-8149-a9cfe3ccada0)

是不是原来的得 通过浏览器获取登陆key 的方式已经不支持了，一会报版本低，一会又是其他错误

**Comment by ForteScarlet** [comment]
#141
